src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random number = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor(),number.nextInt(100));
```

Comment:
ColorSupplier should be class field, like Random
100 should be class constant
don't forget to format your code, Ctrl Alt L shortcut in IDEA for windows will auto format for you